ui-grid/ui-grid.d.ts

```diff
@@ -1512,6 +1512,10 @@ declare namespace uiGrid {
              */
             expandableRowHeight?: number;
             /**
+             * reference to the parent grid scope (the parent scope of the sub-grid element)
+             */
+            expandableRowScope?: Object;
```

Comment:
I just checked it in our source, and found that we can use the suggested types i.e., `ng.IScope | Object;`